boards/x86/qemu_x86/qemu_x86_lakemont.yaml

```diff
@@ -0,0 +1,11 @@
+identifier: qemu_x86_lakemont
+name: QEMU Emulation for X86 (Lakemont)
+type: qemu
+simulation: qemu
+arch: x86
+toolchain:
+  - zephyr
+  - xtools
+  - llvm
+testing:
+  default: true
```

Comment:
make this for now just run the kernel tests

```
  only_tags:
     - kernel
```

Reply:
Added.